### PR TITLE
Create edits.md

### PR DESCRIPTION
**Pull request by b6894**
Here are my suggested edits for your resume.

### CODE REVIEW

**Comment by wilson6747** [comment]
# %%# imports dataimport pandas as pd import altair as altimport numpy as np from sklearn.model_selection import train_test_splitfrom sklearn import treefrom sklearn.ensemble import GradientBoostingClassifierfrom sklearn import metrics # %%# 1 - drops n/a's without using replace or dropnabase_url = 'https://github.com/byuidatascience/data4missing/'flights_path = 'raw/master/data-raw/flights_missing/flights_missing.json'url_flights = base_url + flights_pathflights = (pd.read_json(url_flights))flights = flights[flights.month != 'n/a'] # %%print(flights.month.head(5).to_markdown()) # %%# 2 - replaces both minutes_Delayed_carrier and minutes_delayed_nas nan values with the mean# minutes_delayed_carrierflights = flights.assign(minutes_delayed_carrier = lambda x: x.num_of_delays_late_aircraft.replace(np.nan, flights.minutes_delayed_carrier.mean()))flights_minutes_stdev_carrier = flights.minutes_delayed_carrier.std()# minutes_delayed_nasflights = flights.assign(minutes_delayed_nas = lambda x: x.num_of_delays_late_aircraft.replace(np.nan, flights.minutes_delayed_nas.mean()))flights_minutes_stdev_nas = flights.minutes_delayed_nas.std() # %%# 3 - converts month column to numeric for ml modelflights.month.replace("January", 1, inplace = True)flights.month.replace("Febuary", 2, inplace = True)flights.month.replace("March", 3, inplace = True)flights.month.replace("April", 4, inplace = True)flights.month.replace("May", 5, inplace = True)flights.month.replace("June", 6, inplace = True)flights.month.replace("July", 7, inplace = True)flights.month.replace("August", 8, inplace = True)flights.month.replace("September", 9, inplace = True)flights.month.replace("October", 10, inplace = True)flights.month.replace("November", 11, inplace = True)flights.month.replace("December", 12, inplace = True)month_average = flights.month.mean() # %%# 3 - output for month to numericprint(flights.month.head(5).to_markdown()) # %%# 4 - convert num_of_delays_carrier to numericflights.num_of_delays_carrier.replace("1500+", 1500, inplace = True)flights['num_of_delays_carrier'] = pd.to_numeric(flights.num_of_delays_carrier)num_of_delays_carrier_average = flights.num_of_delays_carrier.mean() # %%# 5 - creates box_plotalt.Chart(flights).mark_boxplot().encode(    x='airport_code',    y='num_of_flights_total').save('box_plot.png') # %%# 6ml_data = flights.sample(500) X_pred = flights.drop(['num_of_delays_total'], axis = 1)y_pred = flights.before1980X_train, X_test, y_train, y_test = train_test_split(    X_pred,     y_pred,     test_size = .33,     random_state = 1936)
